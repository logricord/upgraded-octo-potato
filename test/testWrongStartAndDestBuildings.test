# Verify program correctly outputs message for non-existent starting and destination buildings.
r
LOR
TDS

# Terminate program
q