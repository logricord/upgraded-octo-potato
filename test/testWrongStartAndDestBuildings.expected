Menu:
	r to find a route
	b to see a list of all buildings
	q to quit

Enter an option ('m' to see the menu): # Verify program correctly outputs message for non-existent starting and destination buildings.
Abbreviated name of starting building: Abbreviated name of ending building: Unknown building: LOR
Unknown building: TDS

Enter an option ('m' to see the menu):
# Terminate program
